terraso_backend/config/settings.py

```diff
@@ -113,11 +113,12 @@
     "SCHEMA": "apps.graphql.schema.schema",
 }
 
+WEB_CLIENT_URL = config("WEB_CLIENT_ENDPOINT", default="")
+AUTH_COOKIE_DOMAIN = config("AUTH_COOKIE_DOMAIN", default="terraso.org")
```

Comment:
Question: Should this contain the subdomain too?

Reply:
This is going to consider all subdomains.